specification/2.0/README.md

```diff
@@ -831,57 +832,108 @@ A skin is instanced within a node using a combination of the node's `mesh` and `
 
 ## Texture Data
 
-**TODO: describe separation of concerns: textures / images / samplers**
+glTF separates texture access into three distinct types of objects: Textures, Images, and Samplers.
 
 ### Textures
 
-**TODO: update to latest revisions**
-
-All textures are stored in the asset's `textures` array. A texture is defined by an image file, denoted by the `source` property; `format` and `internalFormat` specifiers, corresponding to the GL texture format types; a `target` type for the sampler; a sampler identifier (`sampler`), and a `type` property defining the internal data format. Refer to the GL definition of `texImage2D()` for more details.
+All textures are stored in the asset's `textures` array. A texture is defined by an image resource, denoted by the `image` property; a `target` type for the sampler; and a sampler index (`sampler`).
+When `sampler` is omitted, a sampler with all-default properties should be used. 
 
 ```json
 {
     "textures": [
         {
-            "format": 6408,
-            "internalFormat": 6408,
-            "sampler": 0,
-            "source": 2,
             "target": 3553,
-            "type": 5121
+            "image": 2,
+            "sampler": 0
         }
     ]
 }
 ```
 
+> **Implementation Note** glTF 2.0 supports only 2D texture targets. Support for other targets is expected in future revisions.
+
 ### Images
 
-**TODO: update to latest revisions**
+Images referred by textures are stored in the `images` array of the asset.
+ 
+Each `image` defines an `image.sources` set of image sources. Type and format of image source are defined by `mimeType` and `internalFormat` specifiers.
+
+An image source can refer to image data either by array of URIs (external or base64-encoded), or by array of Buffer Views.
+
+> **Implementation Note** glTF 2.0 requires these arrays to contain no more than one element. This restriction could be lifted in future to support more texture targets.  
+ 
+First image pixel corresponds to the lower left corner of the image.
+
+> **Implementation Note**: With WebGL API, the first pixel transferred from the `TexImageSource` (i.e., HTML Image object) to the WebGL implementation corresponds to the upper left corner of the source. To achieve correct rendering, WebGL runtimes must flip Y axis by enabling `UNPACK_FLIP_Y_WEBGL` flag. This behavior was changed from glTF 1.0.
```

Comment:
Once this is merged, please update https://github.com/KhronosGroup/glTF/issues/605 with changes like this, especially this one since it is breaking but the syntax doesn't change.